java/client/src/main/java/glide/api/commands/HashBaseCommands.java

```diff
@@ -225,7 +225,7 @@ public interface HashBaseCommands {
     /**
      * Returns all field names in the hash stored at <code>key</code>.
      *
-     * @see <a href="https://valkey.io/commands/hkeys/">redis.io</a> for details
+     * @see <a href="https://valkey.io/commands/hkeys/">redis.io</a> for details.
```

Comment:
don't have to change this now necessarily as I imagine this has to be updated in other spots too, but we should do it at some point
```suggestion
     * @see <a href="https://valkey.io/commands/hkeys/">valkey.io</a> for details.
```